### ISSUE TITLE
Bug: Hotkeys with multiple Modifiers are not shown.

### ISSUE THREAD

**Issue by chrisgrieser** [AlexBieg/obsidian-better-command-palette#4]
I have a bunch of hotkeys with multiple modifiers, e.g. `⌘⌥⇧⌃ T` and they are not shown with the Better Command Palette, even though they are displayed normally with the normal Command Palette. 

https://user-images.githubusercontent.com/73286100/151210025-87324c4a-b11c-4f41-9018-a81631012ef1.mov

**Comment by AlexBieg** [comment]
Yup, definitely a bug.  Might be parsing the hotkeys wrong for some commands. Out of curiosity, are you seeing any hotkeys?

Disclaimer: I'm employed full time so this only gets attention after work. If you need/want this ASAP I'm happy to review a PR 😄 Otherwise I'll get to it as soon as I can.

**Comment by chrisgrieser** [comment]
No worries, I work full time, too, and I am also working on this stuff after work. No need to get everything done instantly, I can survive a few more days with the normal command palette 😝 

If I had an idea what would cause this, I'd make a PR, but I unfortunately have no idea. 😕
